doc/sphinx/source/howto/update.rst

```diff
@@ -39,10 +40,11 @@ To use this version of counting,
 Update in nGraph
 ================
 
-We use the same approach with nGraph. During training, we include all
-the weights as arguments to the training function and return the
-updated weights along with any other results. If we are doing a more
-complex form of training, such as using momentum, we would add the
-momementum tensors are additional arguments and add their updated
-values as additional results. The simple case is illiustrated in the
-trainable model how to.
+In working with nGraph-based construction of graphs, updating takes 
+the same approach. During training, we include all the weights as 
+arguments to the training function and return the updated weights 
+along with any other results. For more complex forms of training, 
+such as those using momentum, we would add the momementum tensors 
+as additional arguments and add their updated values as additional 
```

Comment:
"include" instead of "add" since there are a lot of "add-" in the sentence already?

Reply:
Done